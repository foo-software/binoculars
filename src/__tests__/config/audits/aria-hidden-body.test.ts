import auditAriaHiddenBody from '../../../config/audits/aria-hidden-body';

describe('aria-hidden-body audit', () => {
  it('should match snapshot', () => {
    expect(auditAriaHiddenBody('en')).toMatchSnapshot();
  });
});
