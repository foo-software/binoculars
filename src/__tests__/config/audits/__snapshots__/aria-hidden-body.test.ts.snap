// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`aria-hidden-body audit should match snapshot 1`] = `
Object {
  "description": "Assistive technologies, like screen readers, work inconsistently when \`aria-hidden=\\"true\\"\` is set on the document \`<body>\`. [Learn more](https://web.dev/aria-hidden-body/).",
  "title": "\`[aria-hidden=\\"true\\"]\` is not present on the document \`<body>\`",
}
`;
